src/span.js

```diff
@@ -107,78 +125,6 @@ export default class Span {
     }
 
     /**
-     * Set an arbitrary key-value string pair that will be carried along the
-     * full path of a trace.
-     *
-     * All spans created as children of this span will inherit the baggage items
-     * of this span.
-     *
-     * Baggage items are copied between all spans, both in-process and across
-     * distributed requests, therefore this feature should be used with care to
-     * ensure undue overhead is not incurred.
-     *
-     * Keys are case insensitive and must match the regular expresssion
-     * `[a-z0-9][-a-z0-9]*`.
-     *
-     * @param {string} key
-     * @param {string} value
-     */
-    setBaggageItem(key, value) {
-        if (API_CONFORMANCE_CHECKS) {
-            if (arguments.length !== 2) {
-                throw new Error('Expected 2 arguments');
-            }
-            if (typeof key !== 'string' || key.length === 0) {
-                throw new Error('Key must be a string');
-            }
-            if (!kKeyRegExp.test(key)) {
-                throw new Error('Invalid trace key');
-            }
-
-            let valueType = typeof value;
-            if (value !== null &&
-                valueType !== 'boolean' &&
-                valueType !== 'number' &&
-                valueType !== 'string') {
-                throw new Error('Trace attribute values can only be basic types');
-            }
-        }
-
-        if (this._imp) {
-            this._imp.setBaggageItem(key, value);
-        }
-        return this;
-    }
-
-    /**
-     * Returns the value for the given baggage item key.
-     *
-     * @param  {string} key
-     *         The key for the given trace attribute.
-     * @return {string}
-     *         String value for the given key, or undefined if the key does not
-     *         correspond to a set trace attribute.
-     */
-    getBaggageItem(key) {
```

Comment:
If documented methods are being removed, let's bump the minor version in `package.json`. 

It might also make sense to have a CHANGELOG or "Versions" section in the README that very briefly documents what the API impact between this version and the prior was - specifically around what the consumer of OpenTracing would need to change in their code to keep everything working.